### PR TITLE
An extension sprocket to read blog rss feeds, and display new entries in chat rooms.

### PR DESCRIPTION
**Pull request by NickJosevski**
From issue #9 - https://github.com/Code52/jibbr/issues/9

Every 90 minutes (configurable via TimeSpan Interval) in BlogFeed.cs, the sprocket will take the list of feeds specified in the host app.config file key="Blogs.FeedList" which are pairs of @twitter handles and http://rss.feed.urls comma delimited. 

Based on the Blogs.HoursOldThreshold app config setting currently set at 48 hours the optimal value for this is double the room notification time (90 minutes), that way a new blog post message should only ever be delivered to the channel no more than twice. This is open for discussion how often it should post duplicate posts, if at all.

### CODE REVIEW

**Comment by NickJosevski** [comment]
lol @ my git newbiness here, most of the commits are fixing things with the repository push process, and not code commits.

File: Jabbot.AspNetBotHost/Jabbot.AspNetBotHost.csproj [hunk 1]
```diff
@@ -16,6 +16,7 @@
     <UseIISExpress>true</UseIISExpress>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\Jabbot\</SolutionDir>
     <RestorePackages>true</RestorePackages>
+    <JSLintSkip>\Resources\coffee-script.js</JSLintSkip>
```
Comment (shiftkey) [inline Jabbot.AspNetBotHost/Jabbot.AspNetBotHost.csproj:19]: Where'd this come from?

Comment (NickJosevski) [inline Jabbot.AspNetBotHost/Jabbot.AspNetBotHost.csproj:19]: I needed this for a JSLint extension in VS, it doesn't have negative repercussions if you don't have the extension in VS.

**Comment by shiftkey** [comment]
@NickJosevski i just jump straight to the diff anyway ;)

**Pull request merged**